HTTP2-Core.package/H2TestingTool.class/class/testH2WebClientGetRequestOnLocalhost.st

```diff
@@ -0,0 +1,16 @@
+testH2WebClient
+testH2WebClientGetRequestOnLocalhost
```

Comment:
Consider renaming this method since it does not only try to connect to localhost